### ISSUE TITLE
`width: fit-content` produces error `Invalid value for `width`

### ISSUE THREAD

**Issue by davidmaxwaterman** [csstree/stylelint-validator#29]
> Clearly describe the bug
I expect this to pass without error or warning:
```
    width: fit-content;
```
or to be easily able to find how to make it pass (if it is somehow nonstandard/etc).

> Which rule, if any, is the bug related to?
Not sure. I just installed stylelint.

> What code is needed to reproduce the bug?

```css
div {
  width: fit-content;
}
```

> What stylelint configuration is needed to reproduce the bug?

```json
{
  "plugins": [
    "stylelint-csstree-validator"
  ],
  "rules": {
    "csstree/validator": true
  }
}
```

> Which version of stylelint are you using?
13.7.2

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?
```
npx stylelint *.css
```

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?
No.

> What did you expect to happen?
"No warnings to be flagged."

> What actually happened (e.g. what warnings or errors did you get)?
```shell
✖  Invalid value for `width`   csstree/validator
```

(ref: https://github.com/stylelint/stylelint/issues/5081)

**Comment by lahmatiy** [comment]
That's a bit tricky. According to [specs and MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/width) `fit-content` should be a function. However, browsers are currently supporting it as a keyword.
I suppose we should reflect it in [csstree patch dictionary](https://github.com/csstree/csstree/blob/master/data/patch.json).
As workaround for now you may extend dictionary like so (make sure you use `1.9.0` version of `csstree/validator`):

```json
{
  "plugins": [
    "stylelint-csstree-validator"
  ],
  "rules": {
    "csstree/validator": {
      "properties": {
        "width": "| fit-content"
      }
    }
  }
}
```

**Comment by davidmaxwaterman** [comment]
Thanks. Yes, that seems to work, and allows me to re-enable stylelint :)

**Comment by davidmaxwaterman** [comment]
(nb, I had to add another one - your w/a worked :) `"height": "| min-intrinsic"`)

**Comment by lahmatiy** [comment]
@davidmaxwaterman You may extend or alter syntax that way. See plugin's [options description](https://github.com/csstree/stylelint-validator#options) for detail.

**Comment by voskresla** [comment]
@lahmatiy Can u explain please where this data.patch.json is used? Does it used by 1.9.0 ?

**Comment by mikenath223** [comment]
@voskresla You can simply add it in `.stylelintrc.json`.

**Comment by lahmatiy** [comment]
> Can u explain please where this data.patch.json is used? Does it used by 1.9.0 ?

@voskresla It used in [CSSTree](https://github.com/csstree/csstree) to alter [mdn-data](https://github.com/mdn/data) syntax definition dictionary (mixing [here](https://github.com/csstree/csstree/blob/1.0/data/index.js)). CSSTree is a backend for stylelint-csstree-validator, and mdn-data + patch are used as syntax definition dictionary (that can be altered by options).

**Comment by lahmatiy** [comment]
Fixed in [2.0.0](https://github.com/csstree/stylelint-validator/releases/tag/v2.0.0)
